fs/accounting/stats.go

```diff
@@ -2,6 +2,7 @@ package accounting
 
 import (
 	"bytes"
+	"container/ring"
```

Comment:
I'm not a big fan of the container/ring package it uses an allocation per element and another allocation for the interface it wedges the element in to.

Using a slice of values is a lot more efficient...

Reply:
Sure, a slice is better in every way.